### PR TITLE
Update roadmap with XMTP v3 info

### PR DESCRIPTION
**Pull request by jhaaaa**
[preview html](https://junk-range-possible-git-roadmap-v3-xmtp-labs.vercel.app/roadmap)

### CODE REVIEW

File: src/pages/roadmap.md [hunk 1]
```diff
@@ -8,34 +8,46 @@ Timelines are estimates and subject to change. For information about delivered f
...
+  - XMTP v2-to-v3 migration plan and timeline available to developers
+  - Group chat in development
 - Support for smart contract wallets
-- Portable preferences in Alpha - blocklists, allowlists, etc.
+- Portable preferences in Alpha, including blocklists, allowlists, and more
 
 ## Decentralization roadmap
 
+Roadmap for the decentralization of the XMTP network and the node software that powers its.
+
```
Comment (jhaaaa) [inline src/pages/roadmap.md:50]: Are lines 53 and 54 the same as lines 44 and 45 above? Or just related?

File: src/pages/roadmap.md [hunk 2]
```diff
@@ -8,34 +8,46 @@ Timelines are estimates and subject to change. For information about delivered f
...
+## Messaging roadmap
+
+Roadmap for the messaging protocol and the SDKs and tools that empower developers to build with XMTP.
 
 ### 2023 Q3
 
+- XMTP v3 in Alpha  
+  - Forward secrecy, post-compromise security, and per-app installation inbox management implementations
+  - CLI tooling for E2E testing by internal team and curious developer partners
+  - Announcement of developer calls that enable the community to keep up with and contribute to XMTP v3’s progress
```
Comment (saulmc) [inline src/pages/roadmap.md:26]:
```suggestion
```

Not a product roadmap item.

File: src/pages/roadmap.md [hunk 3]
```diff
@@ -8,34 +8,46 @@ Timelines are estimates and subject to change. For information about delivered f
...
 - Onchain transaction reference content type
 - React SDK in Production
 - React Native SDK in Production (currently in Beta)
 - Android SDK in Production (currently in Beta)
-- XMTP v3 in Alpha
 
 ### 2023 Q4
 
-- XMTP v3 in Beta
+- XMTP v3 in Beta
```
Comment (saulmc) [inline src/pages/roadmap.md:34]: This timeline was developed prior to our org changes. @bwcDvorak do we still feel good about it? 😄

File: src/pages/roadmap.md [hunk 4]
```diff
@@ -8,34 +8,46 @@ Timelines are estimates and subject to change. For information about delivered f
...
 
-- XMTP v3 in Beta
+- XMTP v3 in Beta  
+  - Portable inbox, installation-level revocable keys, and sealed sender implementations
+  - XMTP v3 Beta features in all XMTP SDKs and available for early developer testing
 
 ### 2024 Q1
 
-- XMTP v3 in Production
+- XMTP v3 in Production
```
Comment (saulmc) [inline src/pages/roadmap.md:40]: ditto @bwcDvorak on timeline

File: src/pages/roadmap.md [hunk 5]
```diff
@@ -8,34 +8,46 @@ Timelines are estimates and subject to change. For information about delivered f
...
 ### 2024 Q1
 
-- XMTP v3 in Production
+- XMTP v3 in Production  
+  - XMTP v3 and SDKs ready for use with production apps
+  - XMTP v2-to-v3 migration plan and timeline available to developers
+  - Group chat in development
 - Support for smart contract wallets
-- Portable preferences in Alpha - blocklists, allowlists, etc.
+- Portable preferences in Alpha, including blocklists, allowlists, and more
```
Comment (saulmc) [inline src/pages/roadmap.md:45]: Is this actually planned @bwcDvorak ?

File: src/pages/roadmap.md [hunk 6]
```diff
@@ -8,34 +8,46 @@ Timelines are estimates and subject to change. For information about delivered f
...
+  - XMTP v3 Beta features in all XMTP SDKs and available for early developer testing
 
 ### 2024 Q1
 
-- XMTP v3 in Production
+- XMTP v3 in Production  
+  - XMTP v3 and SDKs ready for use with production apps
+  - XMTP v2-to-v3 migration plan and timeline available to developers
+  - Group chat in development
 - Support for smart contract wallets
```
Comment (saulmc) [inline src/pages/roadmap.md:44]: My understanding is that v3 sets the stage for this but it is not yet resourced the way group chat is.

**Review by saulmc — COMMENTED** [review]
@bwcDvorak lotta questions for you on this one! No expectation that you would have answers yet but good indication of what needs clarifying.

File: src/pages/roadmap.md (hunk 2 above)
Comment (jhaaaa) [reply src/pages/roadmap.md:26]: Along these lines, for comms about the release, Esra'a, Rochelle, and I were wondering if these dev calls are still in the plans? When might they begin? Who will lead them?

File: src/pages/roadmap.md (hunk 3 above)
Comment (bwcDvorak) [reply src/pages/roadmap.md:34]: Yes - it's an aggressive goal but one Naomi and I very much want to push for.

File: src/pages/roadmap.md (hunk 4 above)
Comment (bwcDvorak) [reply src/pages/roadmap.md:40]: Yep! Aggressive goal but doable in our opinion.

File: src/pages/roadmap.md (hunk 5 above)
Comment (bwcDvorak) [reply src/pages/roadmap.md:45]: researching

**Comment by jhaaaa** [comment]
Closing for now. More updates to come.

**Pull request closed without merging**